app/submitter/convert_payload_0_0_2.py

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
Super minor, but are the 2 extra lines needed?

Reply:
Huh, that's slipped through from the original PR, I'll sort it out and push it with any other changes which get requested